odxtools/comparamspec.py

```diff
@@ -3,15 +3,17 @@
 from typing import TYPE_CHECKING, Any, Dict, List, Optional
 from xml.etree import ElementTree
 
+from odxtools.specialdatagroup import SpecialDataGroup
```

Comment:
```suggestion
from .specialdatagroup import SpecialDataGroup
```

(you should re-run the reformatting script after this)